src/platform/src/Bridge/OpenAi/DallE/ModelClient.php

```diff
@@ -23,18 +24,24 @@
  *
  * @author Denis Zunke <denis.zunke@gmail.com>
  */
-final readonly class ModelClient implements ModelClientInterface
+final readonly class ModelClient extends AbstractOpenAiModelClient implements ModelClientInterface
 {
+    private string $baseUrl;
+
+    /**
+     * @param AbstractOpenAiModelClient::REGION_EU|AbstractOpenAiModelClient::REGION_US|null $region
```

Comment:
```suggestion
     * @param self::REGION_*|null $region
```

^ Would work as well